api/api/V1/Station.ts

```diff
@@ -152,6 +153,68 @@ export default function (app: Application, baseUrl: string) {
     }
   );
 
+  /**
+   * @api {post} /api/v1/stations/recordings-count Get count of recordings for multiple stations using their ids in the request body
+   * @apiName GetRecordingsCountForStations
+   * @apiGroup Station
+   * @apiDescription Get the count of recordings associated with multiple stations using their ids in the request body
+   *
+   * @apiUse V1UserAuthorizationHeader
+   *
+   * @apiUse V1ResponseSuccess
+   * @apiInterface {apiSuccess::ApiRecordingsCountResponseSuccess} counts Number of recordings associated with each station
+   * @apiUse V1ResponseError
+   *
+   * @apiBody {Number[]} stationIds Array of station IDs
+   * @apiBody {String} [from-date] Date the station should be active from
+   */
+  app.post(
+    `${apiUrl}/recordings-count`,
+    extractJwtAuthorizedUser,
+    validateFields([
+      body("stationIds")
+        .isArray({ min: 1 })
+        .withMessage("At least one station ID is required"),
+      query("view-mode").optional().equals("user"),
```

Comment:
We don't seem to use view-mode at all in the following logic, so I guess this end-point doesn't really work in super-user mode to view anyone's station recording counts?